docs/README.md

```diff
@@ -37,20 +37,20 @@ The `add-to-cart-button` is a block responsible for adding products in the [Mini
+| `isOneClickBuy`           | `boolean` | Determines whether the user should be redirected to the checkout page (`true`) or not (`false`) when the Add to Cart button is clicked.                                                                                                                                                                                                                           | `false`                                                                                       |
+| `customOneClickBuyLink`   | `string`  | Defines the link to which users will be redirected when the Add to Cart button is clicked, and the `isOneClickBuy` prop is set to `true`.                                                                                                                                                                                                                         | `/checkout/#/cart`                                                                            |
+| `customToastUrl`          | `string`  | Defines the link to which users will be redirected when the toast (pop-up notification displayed when adding an item to the minicart) is clicked.                                                                                                                                                                                                                 | `/checkout/#/cart`                                                                            |
+| `text`                    | `string`  | Defines a custom text message to be displayed on the Add to Cart button.                                                                                                                                                                                                                                                                                          | `Add to cart` _(automatic translation will be applied following your store default language)_ |
+| `unavailableText`         | `string`  | Defines a custom text message to be displayed on the Add to Cart button when a product is unavailable.                                                                                                                                                                                                                                                            | `Unavailable` _(automatic translation will be applied following your store default language)_ |
+| `customPixelEventId`      | `string`  | Defines the `id` for the event that the button will send upon user interaction.                                                                                                                                                                                                                                                                                   | `undefined`                                                                                   |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| - | - | - | - |
| `onClickBehavior` | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page`, `add-to-cart`, and `ensure-sku-selection` (if multiple SKUs are available, users will be redirected to the product page to select the desired one. If the product only has 1 SKU available, it will be automatically added to the cart when the button is clicked). | `add-to-cart` |
| `onClickEventPropagation` | `enum` | Controls whether the 'onClick' event, triggered upon user clicks, should propagate to the parent elements of the page. Possible values are: `disabled` and `enabled`. | `disabled` |
| `isOneClickBuy` | `boolean` | Determines whether the user should be redirected to the checkout page (`true`) or not (`false`) when the **Add to Cart** button is clicked. | `false` |
| `customOneClickBuyLink`   | `string`  | Defines the link to which users will be redirected when the **Add to Cart** button is clicked, and the `isOneClickBuy` prop is set to `true`. | `/checkout/#/cart` |
| `customToastUrl` | `string`  | Defines the link to which users will be redirected when the toast (pop-up notification displayed when adding an item to the minicart) is clicked. | `/checkout/#/cart` |
| `text` | `string`  | Defines a custom text message to be displayed on the **Add to Cart** button. | `Add to cart` _(automatic translation will be applied following your store default language)_ |
| `unavailableText` | `string`  | Defines a custom text message to be displayed on the **Add to Cart** button when a product is unavailable. | `Unavailable` _(automatic translation will be applied following your store default language)_ |
| `customPixelEventId` | `string`  | Defines the `id` for the event that the button will send upon user interaction.                                                                                                                                                                                                                                                                                   | `undefined` |
```